content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -29,6 +29,19 @@ Kubernetes, on Linux:
 - [Docker Engine](#docker)
 - [Mirantis Container Runtime](#mcr)
 
+{{< note >}}
+Dockershim, the portion of code in Kubernetes that provided direct
+integration with Docker in prior releases, was removed from Kubernetes
+version 1.24. This removal was announced as a [deprecation in Kubernetes v 1.20](
+/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
+You can check out this [documentation](
+/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/)
+to understand how this deprecation might affect you. For migrating from
+dockershim you can follow [this migration guide](
+/docs/tasks/administer-cluster/migrating-from-dockershim/)
+to migrate from dockershim.
```

Comment:
If this text is present, the PR must target the _dev-1.24_ branch and not _main_.

/hold
until that discrepancy is dealt with.

Reply:
@sftim done

Reply:
Thanks
/unhold

however: we may well want a similar-but-different change targeting _main_, so that we can blog about this before the release of v1.24

This PR can still go ahead, I think.